toqito/states/ghz.py

```diff
@@ -65,6 +65,11 @@ def ghz(dim: int, num_qubits: int, coeff: list[int] | None = None) -> np.ndarray
     """
     if coeff is None:
         coeff = np.ones(dim) / np.sqrt(dim)
+    else:
+        coeff = np.asarray(coeff)
+        norm = np.linalg.norm(coeff)
+        if not np.isclose(norm, 1.0):
+            coeff = coeff / norm
```

Comment:
could you please add a unit test where a user provides non-normalized coefficients?